From 44f988df9d08873f0e3fd19adb7c062449db9bf3 Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Wed, 12 Apr 2017 09:49:56 +0800
Subject: [PATCH] arm: configs: linux: enable wifi load driver when boot kernel

if we don't enable it, driver init order might be wrong

Change-Id: I0d75b6fcdb762a77f8c27111da642cdba255fc4e
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index 64815dce4888..c86ac23036dc 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -193,7 +193,9 @@ CONFIG_HOSTAP_FIRMWARE_NVRAM=y
 CONFIG_RT2X00=y
 CONFIG_RT2800USB=y
 CONFIG_WL_ROCKCHIP=y
+CONFIG_WIFI_LOAD_DRIVER_WHEN_KERNEL_BOOTUP=y
 CONFIG_AP6XXX=y
+CONFIG_RTL8188EU=y
 CONFIG_MWIFIEX=y
 CONFIG_MWIFIEX_SDIO=y
 # CONFIG_INPUT_MOUSEDEV is not set
-- 
2.35.3

